modules/quickstart/pages/network-quickstart.adoc

```diff
@@ -131,26 +139,189 @@ To check your connection to the {IC} network:
+  "ic_api_version": "0.17.0"  "impl_hash": "f6ccc304526a28bbb91d0603e8c1ba549847eb0524351a822c5ad52ac941e207"  "impl_version": "0.1.0"  "root_key": [48, 129, 130, 48, 29, 6, 13, 43, 6, 1, 4, 1, 130, 220, 124, 5, 3, 1, 2, 1, 6, 12, 43, 6, 1, 4, 1, 130, 220, 124, 5, 3, 2, 1, 3, 97, 0, 181, 243, 155, 255, 186, 159, 221, 212, 218, 49, 250, 130, 50, 177, 148, 34, 24, 191, 102, 20, 93, 48, 182, 174, 80, 183, 210, 201, 0, 152, 175, 34, 176, 99, 112, 129, 94, 138, 24, 126, 210, 238, 209, 29, 65, 35, 149, 187, 0, 4, 52, 151, 90, 147, 7, 234, 23, 130, 9, 192, 108, 253, 219, 228, 119, 217, 93, 155, 186, 165, 182, 135, 219, 232, 158, 45, 235, 113, 37, 221, 105, 153, 203, 205, 133, 25, 109, 115, 164, 248, 120, 48, 230, 167, 118, 134]
 }
 ....
 
+[[net-ledger-id]]
+== Confirm your developer identity and ledger account
+
+All ICP token transactions are recorded in a ledger canister running on the Internet Computer.
+Before you can transfer any ICP tokens you hold in your ledger account, you need to send a secure and properly-signed message that verifies your identity to the ledger and authorizes your developer identity to complete the transaction.
+
+Depending on how you have set up custody for holding your ICP tokens, the hardware, software, and steps required to connect to the ledger and complete a transaction can vary.
+For example, you might connect to the ledger and start a transaction from a hardware wallet, using a hardware security module (HSM) appliance, through the Network Nervous System (NNS) front-end application, or using the {sdk-short-name} `+dfx+` command-line interface.
+Each approach presents a different interface for signing and sending messages to the ledger and representing your identity as an account holder.
+
+=== About your developer identity
+
+The first time you use the {sdk-short-name}, the `+dfx+` command-line tool creates a `+default+` developer identity for you. This identity is represented by a *principal* data type and a textual representation of the principal often referred to as your *principal identifier*.
+This representation of your identity is similar to a Bitcoin or Ethereum address.
+
+However, the principal associated with your developer identity is typically not the same as your *account identifier* in the ledger. The principal identifier and the account identifier are related—both provide a textual representation of your identity—but they use different formats.
```

Comment:
Maybe give examples to emphasize which is which?

Reply:
I see them now in the steps...probably better there anyway, because folks might skip right to the procedure.